packages/interactivity/src/proxies/test/store-proxy.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * Internal dependencies
+ */
+import { proxifyStore, proxifyState } from '../';
+import {
+	setScope,
+	resetScope,
+	setNamespace,
+	resetNamespace,
+	getContext,
+} from '../../hooks';
+
+describe( 'interactivity api - store proxy', () => {
+	describe( 'get', () => {
```

Comment:
Maybe like this?

```suggestion
describe( 'interactivity api', () => {
	describe( 'store proxy', () => {
```

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/62734/commits/75d088908bd5e8782df88b61215151ea1e43de4f.